src/main/webapp/styles/addrecs.css

```diff
@@ -0,0 +1,88 @@
+/* --- Formatting for Popup Box --- */
+
+.popup .overlay {
+  background: rgba(0, 0, 0, 0.3);
+  display: none;
+  height: 100%;
+  left: 0px;
+  position: fixed;
+  top: 0px;
+  width: 100%;
+  z-index: 0;
+}
+
+.popup .popup-content {
+  background: #fff;
+  border-color: #000;
+  border-image: initial;
+  border-style: solid;
+  border-width: initial;
+  box-sizing: border-box;
+  color: #000;
+  height: 534px;
+  left: 75%;
+  margin: 10px;
+  padding: 20px;
+  position: absolute;
+  text-align: center;
+  top: 65%;
+  transform: translate(-50%, -50%) scale(0);
+  width: 450px;
+  z-index: 2;
+}
+
+/* Close Button [X] for the Popup */
+.popup .close-btn {
+  background: #222;
+  border-radius: 50%;
+  color: #fff;
+  cursor: pointer;
+  font-size: 25px;
+  font-weight: 600;
+  height: 30px;
+  line-height: 30px;
+  position: absolute;
+  right: 20px;
+  text-align: center;
+  top: 20px;
+  width: 30px;
+}
+
+.popup.active .overlay {
+  display: block;
+}
+
+.popup.active .popup-content {
+  transform: translate(-50%, -50%) scale(1);
+  transition: all 300ms ease-in-out 0s;
+}
+
+/* Add Rec Form Elements */
+
+label {
+  display: inline-block;
+  padding-right: 5px;
+}
+
+textarea {
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  font-size: 15px;
+  resize: none;
+}
+
+.radio-buttons {
+  margin: 10px;
+}
+
+/* Submit Button for Add Rec Form */
+input[type="submit"] {
```

Comment:
I have a temporary button class style in common.css (in my branch).
 we might want to make all the buttons have the same style using the style in common.css later when we all merge it

Reply:
Yes, I think this will be an issue to resolve when we can merge it. I think we will still need some custom button styles, for different purposes, but an overall look will be important.